pinecone/core/openapi/db_data/api/bulk_operations_api.py

```diff
@@ -89,63 +89,43 @@ def __cancel_import(self, id, **kwargs):
             kwargs["id"] = id
             return self.call_with_http_info(**kwargs)
 
-        self.cancel_import = _Endpoint(
+        self.cancel_bulk_import = _Endpoint(
             settings={
                 "response_type": ({str: (bool, dict, float, int, list, str, none_type)},),
                 "auth": ["ApiKeyAuth"],
                 "endpoint_path": "/bulk/imports/{id}",
-                "operation_id": "cancel_import",
+                "operation_id": "cancel_bulk_import",
```

Comment:
I thought we were _not_ using the word "bulk" in the clients anymore wrt to this endpoint? https://pinecone-io.slack.com/archives/C05MYEX7T2L/p1728935415605249?thread_ts=1728935400.761069&cid=C05MYEX7T2L

cc @austin-denoble

Reply:
(Looks like it's not used in `apis` either: e.g. https://github.com/pinecone-io/apis/blob/main/static/2024-10/db_data_2024-10.oas.yaml#L76)

Reply:
This is generated code, so not really publicly exposed. I think this is fine, there's a similar story in other SDKs since the spec contains "bulk" in a few places.